packages/vscode-ui5-language-assistant-bas-ext/scripts/copy.js

```diff
@@ -0,0 +1,30 @@
+const { copySync, readdirSync } = require("fs-extra");
+const { join } = require("path");
+
+const getBaseSrc = () =>
```

Comment:
why are these "constants" recomputed each time they are invoked? (wrapper in a function).

The results is always the same, so a simple const would suffice (without the lambda wrapper)